react-native/android/src/main/java/io/realm/react/RealmReactModule.java

```diff
@@ -58,7 +65,18 @@ public String getName() {
         if (!isContextInjected()) {
             startWebServer();
         }
-        return Collections.EMPTY_MAP;
+
+        List<String> hosts;
```

Comment:
You're right, we only need just do this in the case where we call `startWebServer()`.